terraform/main.tf

```diff
@@ -431,8 +431,81 @@ provider "helm" {
   }
 }
 
+# Prometheus alertmanager must be hooked up to VictorOps to deliver alerts,
+# which requires an API key. We don't want to check that key into git, so
+# similarly to the batch signing and packet decryption keys, we make a dummy
+# k8s secret here and fill in a real value later. See terraform/README.md.
+resource "kubernetes_secret" "prometheus_alertmanager_config" {
+  metadata {
+    name = "prometheus-alertmanager-config"
+    # Prometheus runs in the default namespace
+    namespace = "default"
```

Comment:
Hmm, does it have to run in the default namespace? Would making a monitoring namespace make sense here?